tests/transaction/test_get_transaction.py

```diff
@@ -0,0 +1,142 @@
+"""
+Provide tests for command line interface's get single transaction command.
+"""
+import json
+
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_single_transaction():
```

Comment:
```suggestion
def test_get_transaction():
```
Change it in all test cases below, please.

Reply:
@anastasiia-bilova, done.